das/daser.go

```diff
@@ -201,6 +214,9 @@ func (d *DASer) catchUpManager(ctx context.Context, checkpoint int64) {
 // catchUp starts a sampling routine for headers starting at the next header
 // after the `from` height and exits the loop once `to` is reached. (from:to]
 func (d *DASer) catchUp(ctx context.Context, job *catchUpJob) (int64, error) {
+	// indicate that DASer is catching up to local chain head
+	atomic.StoreUint64(&d.state.caughtUp, 0)
```

Comment:
I don't really feel comfortable with dealing with literals here. How to improve:

1. Use constants instead of literals.
2. Even better, hide the setting/getting behind an interface that uses the appropriate types (i.e. `bool` in this case).